tests/requirements.txt

```diff
@@ -537,10 +537,11 @@ pytest-metadata==3.1.1 \
     # via
     #   -r requirements.txt
     #   pytest-html
-pytest-profiling==1.7.0 \
-    --hash=sha256:93938f147662225d2b8bd5af89587b979652426a8a6ffd7e73ec4a23e24b7f29 \
-    --hash=sha256:999cc9ac94f2e528e3f5d43465da277429984a1c237ae9818f8cfd0b06acb019
-    # via -r requirements.txt
+# Broken on python version 3.13+
```

Comment:
can you delete this dependency from requirements.txt if we're not using it, then run the generate command at the top of the file https://github.com/nginxinc/kubernetes-ingress/pull/6651/files#diff-27c5febcc9ce93e8e5341f263b7d68962733b8a2a86c583723d025a1db39d7a3R540-R543